bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/src/main/java/com/google/cloud/bigtable/mapreduce/validation/BigtableSyncTableJob.java

```diff
@@ -0,0 +1,247 @@
+ * table only. BigtableSyncTableJob extends functionality to provide connectivity to Bigtable as a
+ * source or target and relies on {@link org.apache.hadoop.hbase.mapreduce.SyncTable} for core
+ * functionality.
+ *
+ * <p>SyncTable is a Mapreduce job that generates hashes on the target table and compares these
+ * hashes with the output from {@Link org.apache.hadoop.hbase.mapreduce.HashTable}. For diverging
+ * hashes, cell-level comparison is performed.
+ */
+public class BigtableSyncTableJob extends SyncTable {
+
+  private static final Log LOG = LogFactory.getLog(BigtableSyncTableJob.class);
+
+  public static final String SOURCE_BT_PROJECTID_CONF_KEY = "sync.table.source.bt.projectid";
+  public static final String SOURCE_BT_INSTANCE_CONF_KEY = "sync.table.source.bt.instance";
+  public static final String SOURCE_BT_APP_PROFILE_CONF_KEY = "sync.table.source.bt.app.profile";
+  public static final String TARGET_BT_PROJECTID_CONF_KEY = "sync.table.target.bt.projectid";
+  public static final String TARGET_BT_INSTANCE_CONF_KEY = "sync.table.target.bt.instance";
+  public static final String TARGET_BT_APP_PROFILE_CONF_KEY = "sync.table.target.bt.app.profile";
+
+  private String sourceBigtableProjectId;
+  private String sourceBigtableInstance;
+  private String sourceBigtableAppProfile;
+
+  private String targetBigtableProjectId;
+  private String targetBigtableInstance;
+  private String targetBigtableAppProfile;
+
+  private static final int NUM_ARGS = 3;
+
+  protected Counters counters;
+
+  public BigtableSyncTableJob(Configuration conf) {
+    super(conf);
+  }
+
+  @Override
+  public int run(String[] args) throws Exception {
+    String[] otherArgs = new GenericOptionsParser(getConf(), args).getRemainingArgs();
+    if (!doCommandLine(otherArgs)) {
+      return 1;
+    }
+
+    Job job =
+        BigtableSyncTableAdapter.setupJobWithBigtable(
+            this,
+            otherArgs,
+            sourceBigtableProjectId,
+            sourceBigtableInstance,
+            sourceBigtableAppProfile,
+            targetBigtableProjectId,
+            targetBigtableInstance,
+            targetBigtableAppProfile);
+    if (!job.waitForCompletion(true)) {
+      LOG.info("Map-reduce job failed!");
+      return 1;
+    }
+    counters = job.getCounters();
+    return 0;
+  }
+
+  /**
+   * SyncTable command line parsing utility
+   *
+   * @param args
+   * @return
+   */
+  private boolean doCommandLine(final String[] args) {
+    String[] toolDefaultArgs = parseBigtableCommandLineArgs(args);
+    if (!BigtableSyncTableAdapter.parseCommandLine(this, NUM_ARGS, toolDefaultArgs)) return false;
+
+    // return false if options and args are not properly set
+    if (!BigtableSyncTableAdapter.verifyRequiredArgsSet(
+        this,
+        args,
+        sourceBigtableProjectId,
+        sourceBigtableInstance,
+        targetBigtableProjectId,
+        targetBigtableInstance)) return false;
```

Comment:
Pl run the formatter.

Reply:
i confirmed, `mvn com.coveo:fmt-maven-plugin:format` emits this formatting